bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooworkspace.py

```diff
@@ -61,8 +61,78 @@ def __setitem__(self, key, value):
                     raise SyntaxError("Invalid syntax")
             else:
                 raise RuntimeError("ERROR importing object named " + key + " another instance with same name already in the workspace and no conflict resolution protocol specified")
+        elif(isinstance(value, dict)):
+            import ROOT
+            import json
+            # Add name attribute to the dictionary, and create the JSON string for the object's JSONTree
+            value["name"] = key
+            variable_attributes = ["min", "max", "value"]
+            is_variable = False
+            # Checks if the object is a variable 
+            for variable_attribute in variable_attributes:
+                is_variable = is_variable or (variable_attribute in value)
```

Comment:
```suggestion
            # Checks if the object is a variable 
            is_variable = not "type" in value
```
I think this would be the cheaper way to check if this is a variable specification, right?

Reply:
Yes, I fixed it now